spec/jekyll/configuration_spec.rb

```diff
@@ -0,0 +1,37 @@
+RSpec.describe(Jekyll::Configuration) do
+  let(:config) { described_class.from({}) }
+
+  describe "#safe_load_file" do
+    it "throws an InvalidFileFormat on a bad extension" do
```

Comment:
```ruby
context "when given a bad extension" do
  it "should throw InvalidFileFormat"
    # Do Work
  end
end
```

So that if the circumstance changes the test can resurface without effort.